msm/msm

```diff
@@ -176,56 +260,74 @@ function install() {
       fi
     fi
     if [[ -f "requirements.txt" ]]; then
-      echo "Installing requirements..."
-      if [[ "${picroft}" == 'false' ]]; then
+      echo "  Installing requirements..."
+      if [[ "${picroft}" == "false" ]]; then
         if [[ "${VIRTUAL_ENV}" =~ .mycroft$ ]] ; then
           if ! pip install -r requirements.txt ; then
-            echo "ERROR: Unable to install requirements for skill ${iskill}!"
+            echo "ERROR: Unable to install requirements for skill '${name}'"
             return 121
           fi
         else
           if workon mycroft ; then
             if ! pip install -r requirements.txt ; then
-              echo "ERROR: Unable to install requirements for skill ${iskill}!"
+              echo "ERROR: Unable to install requirements for skill '${name}'"
               deactivate mycroft
               return 121
             fi
           else
-            echo "ERROR: Unable to activate mycroft virtualenv!"
+            echo "ERROR: Unable to activate Mycroft virtualenv, requirements not installed."
```

Comment:
Should this be small letters since the Mycroft virtualenv name is actually `mycroft`

Reply:
Fixed